src/dealDashboard/dealInfo/dealInfo.html

```diff
@@ -0,0 +1,5 @@
+<template>
+  <pcard>
```

Comment:
should the whole template be in a section?

Reply:
updated. I used the component itself as the container. No need for an additional element